operations/patch_util.go

```diff
@@ -416,6 +417,26 @@ func gitLog(base, ref string) (string, error) {
 	return gitCmd("log", args...)
 }
 
+func gitCommitMessages(base, ref string) (string, error) {
+	args := []string{"--pretty=format:%B", fmt.Sprintf("%s@{upstream}..%s", base, ref)}
```

Comment:
Strange question: The first `%` is rendered in red in the GitHub UI. Is it different from the others?

Reply:
discussed in person. 
Looks okay.